File: fizzbuzz.py
#!/usr/bin/python2

# Language:       Python 2
# Web site:       http://python.org/
# Last tested on: Ubuntu 20.04
# Requires:       The "python" package is pre-installed

for i in xrange(1,101):
    if i % 15 == 0:
        print "FizzBuzz"
    elif i % 3 == 0:
        print "Fizz"
    elif i % 5 == 0:
        print "Buzz"
    else:
        print i

File: fizzbuzz.pl
#!/usr/bin/perl

# Language:       Perl
# Web site:       https://www.perl.org/
# Last tested on: Ubuntu 24.04.2 LTS
# Requires:       The "perl" package is pre-installed

use strict;
use warnings;

for (1..100) {
    my $s = '';
    $_%3 or $s.='Fizz';
    $_%5 or $s.='Buzz';
    $s or $s.=$_;
    print "$s\n";
}
